### PR TITLE
[IOPID-1400] Add opex dashboard for "api_backend"

### PR DESCRIPTION
**Pull request by lucacavallaro**
#### List of Changes

Add Azure Dashboard to monitor `ValidateProfileEmail` endpoint

#### Motivation and Context

This dashboard enables us to monitor the validation e-mail endpoint, that was recently modified for the unique-email-enforcement feature

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by gquadrati — CHANGES_REQUESTED** [review]
Add GH actions

**Review by gquadrati — CHANGES_REQUESTED** [review]
We need to add the TF .identity folder and setup it. Also we need to add GH actions.

File: .identity/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Local .terraform directories
```
Comment (Krusty93) [inline .identity/.gitignore:1]: I wouldn't create another .gitignore but instead I'd add tf things in the main file

File: .identity/00_main.tf [hunk 2]
```diff
@@ -0,0 +1,32 @@
+terraform {
+  required_version = ">=1.3.0"
```
Comment (Krusty93) [inline .identity/00_main.tf:2]: `1.6.0`

File: .identity/00_main.tf.ci
Comment (Krusty93) [inline .identity/00_main.tf.ci:1]: Can be deleted

**Review by Krusty93 — CHANGES_REQUESTED** [review]

**Review by Krusty93 — CHANGES_REQUESTED** [review]
Most of the same comments I already wrote [here](https://github.com/pagopa/io-web-profile-backend/pull/32/files)

**Review by gquadrati — APPROVED** [review]

**Pull request merged**